src/wazuh_modules/vulnerability_scanner/qa/test_data_policy/003/expected_004.out

```diff
@@ -1,4 +1,4 @@
 [
     "Vulnerability scanner module started",
-    "Event type: 8 processed"
+    "Event type: 7 processed"
```

Comment:
Reason to change this?

Reply:
Sometimes this test passes it doesn't

This is due to the fact that event queues are not cleaned in between tests

Reply:
I've added a cleanup for the event queues in between executions